docs/StardustDocs/topics/ColumnSelectors.md

```diff
@@ -127,42 +94,39 @@ df.select { name.allDfs() }
 ```
 
 </tab>
-<tab title="KProperties">
+<tab title="Strings">
 
 ```kotlin
 // by column name
-df.select { it[Person::name] }
-df.select { (Person::name)() }
-df.select { col(Person::name) }
+df.select { it["name"] }
 
 // by column path
-df.select { it[Person::name][Name::firstName] }
-df.select { Person::name[Name::firstName] }
+df.select { it["name"]["firstName"] }
+df.select { "name"["firstName"] }
 
 // with a new name
-df.select { Person::name named "Full Name" }
+df.select { "name" named "Full Name" }
 
 // converted
-df.select { Person::name[Name::firstName].map { it.lowercase() } }
+df.select { "name"["firstName"]<String>().map { it.uppercase() } }
 
 // column arithmetics
-df.select { 2021 - (Person::age)() }
+df.select { 2021 - "age"<Int>() }
```

Comment:
honestly need to say, that Person::age was better than "age"<Int>

Reply:
The PR is a bit unclear in this part. In fact, it's a very bad "diff". In this file after update of korro KProperties API tab was removed (we discussed it with Jolan). And "String API" tab was already there

Reply:
There has also been some shifting around, that's where the clashes come from